### DISCUSSION TITLE
File Multiselect

### DISCUSSION THREAD

**Discussion by BlueFalconHD** [charmbracelet/gum#189] [Ideas]
This is probably a simple feature, but being able to use the newly implemented file tool to pick multiple files would be nice. Maybe a feature to filter out certain files or change how results are seperated. Also, being able to add a custom row of stats, such as the status of that file via git.

**Comment by lbdyck** [comment]
Would also be nice to be able to have the `..` directory show so that one can navigate up the tree instead of down.
